samples/drivers/mbox/boards/lpcxpresso55s69_cpu0.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+        /* Delete ipc chosen property where old IPM mailbox driver bellow is
+		 * configured.
+		 */
+		/delete-property/ zephyr,ipc;
+	};
+
+    soc {
+		/* Delete IPM Driver node nxp,lpc-mailbox */
+		/delete-node/ mailbox@8b000;
+
+		/* Attach MBOX driver to Mailbox Unit */
+        mbox:mailbox0@5008b000 {
+			compatible = "nxp,mbox-mailbox";
+			reg = <0x5008b000 0xEC>;
+			interrupts = <31 0>;
+			rx-channels = <4>;
+			#mbox-cells = <1>;
+			status = "okay";
+		};
+	};
```

Comment:
the indentation looks odd (for every `.overlay` file in this commit)

Reply:
Thanks, The sneaky whitespacess. Resolved.